ci-operator/step-registry/aws/provision/route53/aws-provision-route53-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
I saw we did not install jq in `ci-operator/step-registry/aws/deprovision/route53/aws-deprovision-route53-commands.sh`, so shall we explicitly install it here?

Reply:
Yes, here it is required to get the hostedzone and change id from the json file.

Reply:
I meant jq should be already installed, if not, `aws-deprovision-route53` ref would failed.

Reply:
Yes, no need to install `jq`.